doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1256,9 +1409,14 @@ A `subject` contains the following subfields:
 A `notification` object contains the following subfields:
 
 + `attributes`: List of attributes to be included in the notification message. If not specified (or empty), all attributes are included
-  in the notification
+  in the notification. It also defines the order in which attributes will appear in notifications when `attrsFormat` "value" is used
+  (see "Notifications formats") section.
 + `callback` : URL pointing to the service which will be invoked when a notification is generated. A NGSIv2
   compliant server must support `http` URL schema, other schemas (e.g. schemas for web sockets) could also be supported.
++ `headers`: a key-map of HTTP headers that will be included in the notification messages.
```

Comment:
In some sense, that's correct. The new fields `header` and `query` can be sent as part of such notification template.